bitnami/fluentd/README.md

```diff
@@ -106,7 +106,9 @@ The following tables lists the configurable parameters of the fluentd chart and
 | `forwarder.serviceAccount.name`                 | The name of the ServiceAccount to create                                                                       | Generated using the `fluentd.fullname` template                                                         |
 | `forwarder.serviceAccount.annotations`          | Additional Service Account annotations (evaluated as a template)                                               | `{}`                                                                                                    |
 | `forwarder.rbac.create`                         | Specify whether RBAC resources should be created and used, allowing the get, watch and list of pods/namespaces | `true`                                                                                                  |
-| `forwarder.rbac.pspEnabled`                     | Specify whether the bundled Pod Security Policy should be created and bound with RBAC                          | `false`                                                                                                  |
+| `forwarder.rbac.pspEnabled`                     | Specify whether the bundled Pod Security Policy should be created and bound with RBAC                          | `false`                                                                                                 |
+| `forwarder.extraInitContainers`                 | Additional init containers to add to the pods (fluentd forwarder)                                              | `[]`                                                                                                    |
+| `forwarder.sidecars`                            | Add additional containers to the pod (fluentd forwarder)                                                       | `[]`                                                                                                    |
 | `forwarder.extraVolumes`                        | Extra volumes                                                                                                  | `nil`                                                                                                   |
 | `forwarder.extraVolumeMounts`                   | Mount extra volume(s),                                                                                         | `nil`                                                                                                   |
```

Comment:
A small petition, could you please remove the comma at the end of the descriptions of `forwarder/aggregator.extraVolumeMounts`? It seems like a typo from a previous change.
```suggestion
| `forwarder.extraVolumeMounts`                   | Mount extra volume(s)                                                                                          | `nil`                                                                                                   |
```